src/api.rs

```diff
@@ -505,37 +450,6 @@ mod tests {
         assert_eq!(retval.unwrap(), 1);
     }
 
-    // assert an invisible message is not readable
-    #[pg_test]
-    fn test_default() {
-        let qname = r#"test_default"#;
```

Comment:
I had to remove this test because it was using `pgmq_read` rust impl, but I assume this is covered in integration tests? Otherwise we can add it there.